components/brave_extension/extension/brave_extension/background/api/cosmeticFilterAPI.ts

```diff
@@ -94,10 +94,14 @@ const tryMigratingLegacyCosmeticFilters = () => {
   })
 }
 
-export const addSiteCosmeticFilter = async (host: string, cssSelector: string) => {
+export const addSiteCosmeticFilter = (host: string, cssSelector: string) => {
```

Comment:
Should this be changing to not async? I know there is a delay when trying to calculate the selector sometimes and I was able to get it to lag the browser quite a bit by doing right click / adding a lot of selectors on a news site